packages/legend-studio/src/models/protocols/pure/v1/transformation/pureProtocol/serializationHelpers/V1_ValueSpecificationSerializer.ts

```diff
@@ -207,13 +207,13 @@ const V1_appliedFunctionModelSchema = createModelSchema(V1_AppliedFunction, {
 const appliedPropertyModelSchema = createModelSchema(V1_AppliedProperty, {
   _type: usingConstantValueSchema(V1_ValueSpecificationType.APPLIED_PROPERTY),
   class: optional(primitive()),
-  property: primitive(),
   parameters: list(
     custom(
       (val) => V1_serializeValueSpecification(val),
       (val) => V1_deserializeValueSpecification(val),
     ),
   ),
+  property: primitive(),
```

Comment:
Let's separate this into another commit (also because changes like this often means we don't have test covering it). We will rebase this PR and not squash it. If this proves to be too painful, we can pass. But let's do that next time!